lib/themes/dosomething/paraneue_dosomething/templates/system/patterns/photo.tpl.php

```diff
@@ -1,17 +1,20 @@
-<figure class="photo -stacked -framed" data-reportback-item-id="<?php print $content->id; ?>">
-  <?php if (isset($content->admin_link)): ?>
-    <div class="admin-edit">
-      <a class="button -secondary" href="<?php print $content->admin_link; ?>"><?php print t('Edit Status'); ?></a>
-    </div>
-  <?php endif; ?>
-  <img src="<?php print $content->media['uri']; ?>" alt="<?php print filter_xss($content->caption); ?>" />
-  <figcaption class="__copy">
-    <p class="photo__caption"><?php print filter_xss($content->caption); ?></p>
-  </figcaption>
+<div class="photo -stacked -framed" data-reportback-item-id="<?php print $content->id; ?>">
+  <figure class=wrapper>
+    <?php if (isset($content->admin_link)): ?>
+      <div class="admin-edit">
+        <a class="button -secondary" href="<?php print $content->admin_link; ?>"><?php print t('Edit Status'); ?></a>
+      </div>
+    <?php endif; ?>
+    <img src="<?php print $content->media['uri']; ?>" alt="<?php print filter_xss($content->caption); ?>" />
+    <figcaption class="__copy">
+      <p class="photo__caption"><?php print filter_xss($content->caption); ?></p>
+    </figcaption>
+  </figure>
+
   <?php if ($content->allow_reactions): ?>
-    <div class="form-actions -inline photo--actions">
+    <ul class="form-actions -inline photo--actions">
```

Comment:
Ooof, not related to this PR but I should have named this `.photo__actions`. 😞 Mind fixing that up as well?